Neos.Neos/Configuration/Caches.yaml

```diff
@@ -2,7 +2,7 @@ TYPO3_Neos_Configuration_Version:
   frontend: Neos\Cache\Frontend\StringFrontend
   backend: Neos\Cache\Backend\FileBackend
 
-TYPO3_Neos_TypoScript:
+Neos_Neos_Fusion:
```

Comment:
I think we should mark this commit / the overall pull request as breaking (!!!) so it appears in the list of breaking changes in the changelog. Because some people may have chosen to configure a custom cache backend (Redis etc.) which will need to be adjusted.